cli/src/main/java/org/jboss/as/cli/impl/BootScriptInvoker.java

```diff
@@ -87,11 +88,6 @@ public void runCliScript(ModelControllerClient client, File file) {
         } catch (Exception ex) {
             try {
                 ROOT_LOGGER.errorProcessingScript(file);
-                StringBuilder errBuilder = new StringBuilder();
```

Comment:
Is it intentional to get rid of this?

Reply:
Yes, the way errors were reported was not nice. The command is now printed to the output and output displayed when an error occurs. Much easier to match the faulty line and the error. The output is not shown if the CLI script execution was successful.